src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/InstructionList.cs

```diff
@@ -116,7 +114,7 @@ public DebugView(InstructionList list)
                 int stackDepth = 0;
                 int continuationsDepth = 0;
 
-                var cookieEnumerator = (debugCookies != null ? debugCookies : new KeyValuePair<int, object>[0]).GetEnumerator();
+                var cookieEnumerator = (debugCookies != null ? debugCookies : Array.Empty<KeyValuePair<int, object>>()).GetEnumerator();
                 var hasCookie = cookieEnumerator.MoveNext();
```

Comment:
I'll skip on this to keep the changelist small; the interpreter code is quite `var`-heavy at the moment. Maybe we can use (or write if non-existent right now) a Roslyn analyzer to perform some rewrites in bulk for cases where `var` is unclear from context?